plugin/pkg/scheduler/algorithmprovider/defaults/defaults.go

```diff
@@ -204,3 +208,22 @@ func defaultPriorities() sets.String {
 		),
 	)
 }
+
+func GetEquivalencePod(pod *api.Pod) interface{} {
+	equivalencePod := EquivalencePod{}
+	podSpec := &(pod.Spec)
+	for _, vol := range podSpec.Volumes {
+		equivalencePod.Volumes = append(equivalencePod.Volumes, vol)
+	}
+	equivalencePod.NodeSelector = podSpec.NodeSelector
+	equivalencePod.Request = predicates.GetResourceRequest(pod)
+	equivalencePod.Ports = predicates.GetUsedPorts(pod)
+	return &equivalencePod
+}
+
+type EquivalencePod struct {
```

Comment:
As I wrote in the previous PR - this is not enough for the pod to actually be equivalent to the other one.
We should probably open an issue for this particular problem and discuss it there.